lapis/cli/simulate.py

```diff
@@ -88,34 +108,84 @@ def cli(ctx, seed, until, log_tcp, log_file, log_telegraf, calculation_efficienc
         click.Choice(list(storage_import_mapper.keys())),
     ),
     default=(None, None, None),
+    multiple=True,
+    help="Tuple of `(storage_file,storage_content_file,storage_type)`",
+)
+@click.option(
+    "--remote-throughput",
+    "remote_throughput",
+    type=float,
+    default=1.0,
+    help="Parameter to set the network bandwidth to remote",
+)
+@click.option(
+    "--filebased_caching",
+    "filebased_caching",
+    is_flag=True,
+    help="Flag to set filebased caching on/off",
+    default=False,
 )
-@click.option("--remote-throughput", "remote_throughput", type=float, default=10)
 @click.option("--cache-hitrate", "cache_hitrate", type=float, default=None)
 @click.pass_context
-def static(ctx, job_file, pool_file, storage_files, remote_throughput, cache_hitrate):
+def static(
+    ctx,
+    job_file,
+    pre_job_rank,
+    machine_ads,
+    job_ads,
+    scheduler_type,
+    pool_files,
+    storage_files,
+    remote_throughput,
+    filebased_caching,
+    cache_hitrate,
+):
     click.echo("starting static environment")
     simulator = Simulator(seed=ctx.obj["seed"])
-    file, file_type = job_file
+    infile, file_type = job_file
     simulator.create_job_generator(
-        job_input=file,
+        job_input=infile,
         job_reader=partial(
             job_import_mapper[file_type],
             calculation_efficiency=ctx.obj["calculation_efficiency"],
         ),
     )
-    simulator.create_scheduler(scheduler_type=CondorJobScheduler)
+    click.echo(
+        "scheduler configuration: \n "
+        "\tscheduler type: {}\n\n"
+        "\tpre job rank: {} \n\n"
+        "\tmachine classads:\n \t{}\n\n"
+        "\tjob classads: {}".format(scheduler_type, pre_job_rank, machine_ads, job_ads)
+    )
```

Comment:
Use f-strings if you immediately interpolate a string.
```suggestion
    click.echo(
        "scheduler configuration: \n "
        f"\tscheduler type: {scheduler_type}\n\n"
        f"\tpre job rank: {pre_job_rank} \n\n"
        f"\tmachine classads:\n \t{machine_ads}\n\n"
        f"\tjob classads: {job_ads}"
    )
```